src/js/components/SelectMultiple/__tests__/SelectMultiple-test.tsx

```diff
@@ -37,8 +37,8 @@ const defaultOptions = [
 ];
 
 const TestManyOptions = () => {
-  const [options, setOptions] = useState(defaultOptions);
-  const [valueMultiple, setValueMultiple] = useState([]);
+  const [options, setOptions] = useState<string[]>(defaultOptions);
```

Comment:
why was the `<string[]>` added here?

Reply:
do we just in to expand on types so we dont need this?

Reply:
I added the Type because of the error I was getting while setting the options.

Reply:
I am not sure if any other Type is left. We need to decide the Types that are required and then make appropriate Changes to support those.
If there's some Trivial Case left which you can point out, I'll add those.

Thanks